### ISSUE TITLE
r4/player... evolving with two different DB in dev_env

### ISSUE THREAD

**Issue by hugurp** [internet4000/radio4000#44]
Radio4000 in dev mode uses: 
- `staging db`
- player-vue, which uses `live db`

Future `add-track` module will also suffer this problem. When being in dev mode it will use sataging db, but when implemented in r4 though npm (for exemple), it will use live db (so if r4 in dev mode there is again a conflict in db)

This gets us into conflicts while devloping, that could lead us to error prone situations.

ex:
- playing a channel that only exists in staging db will result in weird player erro (as channel & track data do not exist on live db, where player is plugged)
- submitting feedback adds data to the feedback, like userChannel, playerChannel, playerTrack
- playing the first track on a channel in staging db often plays a track which is not the first in the player, or tracks that do not exist anylonger on live db

Solution?

- maybe when in development mode the player, on r4, could plug to the dev db
- both r4 and r4-player could use the same firebase instance no matter what env they're in

**Comment by oskarrough** [comment]
An easy workaround could be this?

- In the Ember app we expose the Firebase `databaseURL` as `window.r4.databaseURL`
- In the player's `store.js` we do

```js
let host = 'https://radio4000.firebaseio.com'
if (window.r4.databaseURL) {
  host = window.r4.databaseURL
}
```

**Comment by oskarrough** [comment]
I tested the above and it does work. Pretty simple. Shall we go for that or do we have something more elaborate?

**Comment by oskarrough** [comment]
Closed by #47
